docs/synapse/userguides/storm_ref_data_mod.rstorm

```diff
@@ -0,0 +1,746 @@
+
+.. _prop-del:
+
+Delete Properties
+-----------------
+
+Operation to delete (fully remove) one or more properties from the specified node(s).
+
+.. WARNING::
+  Storm syntax to delete properties has the potential to be destructive if executed following an incorrect, badly formed, or mistyped query. Users are **strongly encouraged** to validate their query by first executing it on its own (without the delete property operation) to confirm it returns the expected nodes before adding the delete syntax. While the property deletion syntax cannot fully remove a node from the hypergraph, it is possible for a bad property deletion operation to irreversibly damage hypergraph pivoting and traversal.
+
+**Syntax:**
+
+*<query>* **[ -:** *<prop>* ... **]**
+
+**Examples:**
+
+*Delete a property:*
+
+
+.. storm-pre:: inet:ipv4=127.0.0.1 [ -:loc ]
+::
+
+    <inet:ipv4> [ -:loc ]
+
+
+*Delete multiple properties:*
+
+
+.. storm-pre:: media:news [ -:author -:summary ]
+::
+
+    <media:news> [ -:author -:summary ]
+
+
+**Usage Notes:**
+
+- Property deletions are performed on the output of a previous Storm query.
+- Storm will delete the specified property / properties for all nodes in the current working set (i.e., all nodes resulting from Storm syntax to the left of the *-:<prop>* statement), **whether those nodes are within or outside of the brackets** unless :ref:`edit-parens` are used to limit the scope of the modifications.
+- Deleting a property fully removes the property from the node; it does not set the property to a null value.
+- Properties which are read-only ( ``'ro' : 1`` ) as specified in the data model cannot be deleted.
+
+.. _node-del:
+
+Delete Nodes
+------------
+
+Nodes can be deleted from a Cortex using the Storm :ref:`storm-delnode` command.
+
+.. _light-edge-add:
+
+Add Light Edges
+---------------
+
+Operation that links the specified node(s) to another node or set of nodes (as specified by a Storm expression) using a lightweight edge (light edge).
+
+See :ref:`light-edge` for details on light edges.
+
+**Syntax:**
+
+*<query>* **[ +(** *<verb>* **)> {** *<storm>* **} ]**
+
+*<query>* **[ <(** *<verb>* **)+ {** *<storm>* **} ]**
+
+.. NOTE::
+  The nodes specified by the Storm expression ( ``{ <storm> }`` ) must either already exist in the Cortex or must be created as part of the Storm expression in order for the light edges to be created.
+
+.. NOTE::
+  The query syntax used to create light edges will **yield the nodes that are inbound to the edit brackets** (that is, the nodes represented by *<query>*).
+
+**Examples:**
+
+*Link the specified FQDN and IPv4 to the media:news node referenced by the Storm expression using a "refs" light edge:*
+
+
+.. storm-pre:: inet:fqdn=woot.com inet:ipv4=1.2.3.4 [ <(refs)+ { media:news=a3759709982377809f28fc0555a38193 } ]
+.. storm-pre:: [inet:fqdn=newsonet.net inet:fqdn=staycools.net inet:fqdn=hugesoft.org inet:fqdn=purpledaily.com +#aka.feye.thr.apt1]
```

Comment:
duplicate line?

Reply:
not a dupe, one is for pre processing and one is for printing the literal